docs/pywbemcligeneraloptions.rst

```diff
@@ -72,7 +70,7 @@ information on each command-group arguments and options):
 * **--output-format/-o** - Output format choice (Default: mof).
   Note that the actual output format may differ from this value because some
   subcommands only allow selected formats. See :ref:`Output formats`.
-* **--use-pull_ops** [``yes``|``no``|``either``] - Determines whether the pull
+* **--use-pull-ops** [``yes``|``no``|``either``] - Determines whether the pull
```

Comment:
The three choices show up in the formatted output with double backquotes. I suggest inserting space around each backquoted string solves that.

Reply:
DONE